optimize/backend/src/main/java/io/camunda/optimize/service/UIConfigurationService.java

```diff
@@ -70,6 +72,8 @@ public UIConfigurationResponseDto getUIConfiguration() {
     uiConfigurationDto.setMaxNumDataSourcesForReport(
         configurationService.getUiConfiguration().getMaxNumDataSourcesForReport());
     uiConfigurationDto.setOptimizeDatabase(ConfigurationService.getDatabaseType(environment));
+    uiConfigurationDto.setValidLicense(isCamundaLicenseValid());
+    uiConfigurationDto.setLicenseType(getLicenseType().getName());
```

Comment:
Do we already have a FE follow up to use these?
Also lets add these fields to our r[est api docs](https://confluence.camunda.com/display/CO/UI+Configuration)

Reply:
We do! Tnose tickets are linked on the epic https://github.com/camunda/product-hub/issues/2359

Added!